core/embed/extmod/modtrezorutils/modtrezorutils-meminfo.h

```diff
@@ -487,6 +493,14 @@ void dump_trezor_vcp(FILE *out, const mp_obj_VCP_t *vcp) {
   fprintf(out, ",\n");
 }
 
+void dump_protomsg(FILE *out, const mp_obj_protomsg_t *value) {
+  print_type(out, "protomsg", NULL, value, false);
+  fprintf(out, ",\n\"shortval\": ");
```

Comment:
`print_type` will print shortval by itself, so this will create a duplicate key in json.
json doesn't mind, I think, but still not great.

shortval is probably not best for this anyway. I'd stick it in `"message_name"`, i don't see any reason to match this attribute to anything else?

Reply:
right, i wasn't sure how exactly is shortval special ... fixed in 88d5f81, please take a look